tests/functional/spec/introduction.spec.js

```diff
@@ -6,7 +6,7 @@ describe("Introduction page", () => {
     await browser.openQuestionnaire(introductionSchema);
   });
 
-  it("Given I start a survey, When I view the introduction page, Then I should be able to see introduction information", async () => {
+  it("Given I start a survey, When I view the introduction page, Then I should be able to see introduction information displayed correctly", async () => {
```

Comment:
Minor (and these and preexisting), but these are slightly ambiguous to me. For your test specifically I would use `When preview content is set on the introduction page, then the content headings should be displayed at the correct level` or something similar? But this has been added to the existing test and may not make sense to have it separate

Reply:
I will separate it out since your test description is on point.